src/foremast/securitygroup/create_securitygroup.py

```diff
@@ -162,10 +199,13 @@ def add_cidr_rules(self, rules):
                 if 'InvalidPermission.Duplicate' in str(error):
                     self.log.debug('Duplicate rule exist, that is OK.')
                 else:
-                    msg = 'Unable to add cidr rules to {}'.format(rules['app'])
+                    msg = 'Unable to add cidr rules to {}'.format(rule.get('app'))
                     self.log.error(msg)
                     raise SpinnakerSecurityGroupError(msg)
 
+        # Tag security group
+        self.add_tags(group_id)
```

Comment:
Is `add_cidr_rules` the right place to call the function? Seems like it might make more sense under `create_security_group` and do it right after `add_cidr_rules`.